src/client/webpackEntry.js

```diff
@@ -1,10 +1,20 @@
+import {APP_WEBPACK_PUBLIC_PATH_DEFAULT} from 'universal/utils/constants';
+
 /*
  * Setup webpack runtime public path loading, used for configuring
  * asset loading from our CDN:
  */
+/* eslint-disable no-underscore-dangle */
+function getWebpackPublicPath() {
```

Comment:
Rather than create a new file, I found it easiest to put this here. I could have made it a one-liner below, but I found this easier to read.